docs/hoon/concepts.md

```diff
@@ -179,31 +180,32 @@ A `mold` is a constructor function (`gate`).  Its sample is any
 noun; its product is a structured noun.  A mold is idempotent;
 `(mold (mold x))` always equals `(mold x)`.
 
-Since there's no such thing as a [`span`](#span) declaration, we use molds
-to define useful spans by example; we also use them to validate
-untrusted network input.
+Because there's no such thing as a [`type`](#type) declaration, 
+we use molds to define useful types by example; we also use them 
+to validate untrusted network input.
 
 ### `face` (named variable)
 
 In a conventional language, we have a scope, environment or
 symbol table.  Declaring a variable, like `var foo: atom`, adds
 the name `foo` to the table with type `atom`.
 
-The Hoon equivalent is `:var  foo  atom`.   But Hoon has a
+The Hoon analogue is `=|(foo atom)`.   But Hoon has a
 homoiconic heap; there is no inscrutable scope or environment.
 There is just the subject, which is one noun.  To "declare a
-variable" is to make a cell `[variable old-subject]`, and use it
-as the subject of the [`twig`](#twig) below.
+variable" is to make a cell `[variable old-subject]`, and use that
+as the subject of the next expression.
 
-So the labe[`twig`](#twig)l `foo` isn't a key in a symbol table; it's in the
-type ([`span`](#span)) of the new value.  It's not a variable named `foo`,
-whose value is of type `atom`; it's a subtree of type `foo:atom`.
+The label `foo` isn't a key in a symbol table; it's stored as a kind 
+of metadata, in the type ([`type`](#type)) of the new value.  It's not 
+a variable named `foo`, whose value is of type `atom`; it's a subtree 
+of type `foo:atom`.
 
 ### <a name="limb">`limb`</a> (attribute or variable reference)
 
 A [`limb`](../twig/limb/limb), like `foo`, is Hoon's equivalent of a variable
-reference.  A limb is a [`twig`](#twig); given a subject [`span`](#span),
-[`mint`](#mint) resolves it to a Nock formula and a product [`span`](#span).
+reference.  A limb is a [`hoon`](#hoon); given a subject [`type`](#type),
+[`mint`](#mint) resolves it to a Nock formula and a product [`type`](#type).
```

Comment:
I think we should also mention that the hoon compiler always resolves a limb to a tree axis lookup, even though there are multiple kinds of limb. Since a face always refers to a specific tree axis within a member of a type, using a face as a limb still resolves to a numeric axis at compile-time.

Reply:
change made